.gitignore

```diff
@@ -7,4 +7,8 @@ ssl/index.txt*
 ssl/serial*
 
 # terraform
-**/*.tfstate*
\ No newline at end of file
+**/*.tfstate*
+
+# IntelliJ IDEA
+.idea/
+microservices-infrastructure.iml
```

Comment:
please remove these and add them in your global ignores file. We don't want to have IDE-specific ignore files in the project.